code/track.ss

```diff
@@ -5,17 +5,46 @@
       (delete-file config.json))
     (with-output-to-file config.json
       (lambda ()
-	(json-write (process-config))))))
+        (json-write 'pretty (process-config))))))
 
+;; Top level helper for make-config
 (define (process-config)
   (map (lambda (x)
          (if (not (eq? (car x) 'exercises))
              x
-             `(exercises . ,(remp (lambda (exercise)
-                                    (memq 'wip (map car exercise)))
-                                  (cdr x)))))
+             (cons 'exercises
+                   (exercises->snake-case
+                    (remp (lambda (exercise)
+                            (memq 'wip (map car exercise)))
+                          (cdr x))))))
        track-config))
 
+;; Top level helper for process config
+(define (exercises->snake-case exercises)
+  (let ((handle (lambda (exercise)
+                  (map format-js-pair exercise))))
+    (map handle exercises)))
+
+;; Helpers for exercises->snake-case
+(define (kebab->snake str)
+  (let ((k->s (lambda (c) (if (char=? c #\-) #\_ c))))
+    (apply string (map k->s (string->list str)))))
+
+(define (sym-bol->str_ing symbol)
```

Comment:
This name is pretty cheeky :smile: 

I think it shows what it does pretty well, and also looks horrible, so no-one will be likely to use it unless they have to :rofl:

If you think it needs to change feel free to let me know your ideas ;)

Reply:
The name is definitely funny. But it's gotta change haha

Reply:
What do you think it should be called?

Reply:
Come to think of it, this should probably be defined in `code/outils.ss`. That, or be locally defined where it's useful.

I'm wondering if there's a chance the underscores don't matter? `configlet lint .` called on un-de-slug-cased json doesn't report anything

Reply:
`symbol->snake-case-string` or something like that for name perhaps

Reply:
Internal to `kebab->snake` I originally named the `k->s` as `-->_` but that was a joke 🤣

Reply:
Regarding your question about whether the snake_case matters.

Since it is getting processed by rails and some of those are definitely rails symbols I think it could cause some problems on the rendering side of the website.

I will investigate further.  If configlet doesn't complain about that then it is probably a bug.

Reply:
We're you talking about the keys?

Reply:
Both actually.

Reply:
Yeah, so the slugs have to be `kebab-case`. Configlet will fail the Travis build otherwise.

The topics list is getting parsed by rails somewhere, and I imagine it is making some assumptions about `snake_case` there.

I will investigate further.

Reply:
changed the name